contents/DashboardBenefitTasksConstants.js

```diff
@@ -305,50 +306,57 @@ const SUBMITTED_EI_TASKS = [
   {
     task: t.viewPaymentInfo,
     taskIcon: '/images/dashboard/ei-view-payment-info-icon.svg',
-    taskLink: '/dashboard',
+    taskLink:
+      'https://srv136.services.gc.ca/sc/msca-mdsc/portal-portail/pro/ei-ae/meiio-mraed/Pages/MyPayments.aspx?prov=6',
```

Comment:
the links in different benefits are the same why not using property instead of hard coding the link? and are doing only English ?